### ISSUE TITLE
Allow having bindings referencing const Properties

### ISSUE THREAD

**Issue by lemirep** [KDAB/KDBindings#29]
At the moment I cannot use a const Property in a binding. 
```
const Property<float> someValue = makeBinding(someExpression);
Property<float> boundValue = makeBinding(someValue); // This does not compile
```
A const Property means it cannot be assigned to again but its value still might change (if the value of the const property is a binding). Therefore, being able to use a const Property in a binding would make sense.

**Comment by LeonMatthesKDAB** [comment]
Hm, this is an issue with the `makeBinding` function and node operators.
These functions currently take a reference to a Property, even though that could be a const reference.

**Comment by LeonMatthesKDAB** [comment]
Closed as of #72
